mappings/net/minecraft/entity/passive/HorseColor.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_5149 net/minecraft/entity/passive/HorseColor
-	FIELD field_23823 VALUES Ljava/util/function/IntFunction;
+	FIELD field_23823 BY_INDEX Ljava/util/function/IntFunction;
```

Comment:
Yeah, let's use ID everywhere for integers. Index is an implementation detail that only made sense when it was an array.